### ISSUE TITLE
[BOOM-1223] [Again!!] Undefined $urls index in ajax rendered template

### ISSUE THREAD

**Issue by b0tm1nd** [PrestaShop/PrestaShop#10638]
**Describe the bug**
The bug described in http://forge.prestashop.com/browse/BOOM-1223 and supposed to be fixed by https://github.com/PrestaShop/PrestaShop/pull/8856 still present in 1.7.4.1 version.

**Additionnal information**
PrestaShop version: 1.7.4.1

**Comment by khouloudbelguith** [comment]
Hi @b0tm1nd,

Could you please add steps to reproduce the issue in a new comment.

Best regards, Khouloud

**Comment by b0tm1nd** [comment]
This is exactly, word to word as described in this comment: http://forge.prestashop.com/browse/BOOM-1223?focusedCommentId=155180&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-155180

**Comment by b0tm1nd** [comment]
Solution posted here http://forge.prestashop.com/browse/BOOM-1223?focusedCommentId=161820&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-161820 using controller override solves the problem.

**Comment by khouloudbelguith** [comment]
@eternoendless @mickaelandrieu what do you think of this issue?

**Comment by b0tm1nd** [comment]
You can put say {$urls.theme_assets} somewhere in your /themes/yourtheme/modules/ps_shoppingcart/ps_shoppingcart.tpl and just load this in the browser https://yourdomain.com/module/ps_shoppingcart/ajax

In debug mode it will give 500 error with undefined $urls, in normal mode it will be just empty in place of the var.

**Comment by khouloudbelguith** [comment]
Hi @b0tm1nd,

Thanks for this clarifications.
I manage to reproduce this issue with PS1.7.4.2
![image](https://user-images.githubusercontent.com/16067358/45893381-51039480-bdcb-11e8-9915-46a88b24b0cf.png)
We will see how to fix it.

Best regards, Khouloud

**Comment by eternoendless** [comment]
Hi there,

As it happens, the original bug had been fixed in the core but not in the ps_shoppingcart module.

**Comment by eternoendless** [comment]
Fixed in v2.0.2

**Comment by sim1-81** [comment]
i've just solved by adding parent::initContent() within my module ajax controller

```
class SomeModuleAjaxModuleFrontController extends ModuleFrontController {

    public function initContent() {
        $this->ajax = true;
        
        parent::initContent();

        /* my code ...... */
   }
}
```
this works fine in my case PS 1.7.5.1
